docs/reference/options.md

```diff
@@ -3525,7 +3525,7 @@ attribute set of attribute set of (INI atom (null, bool, int, float or string) o
 
 
 ## services.postgres.createDatabase
-Create a database named like current user on startup.
+Create a database named like current user on startup. Only applies when initialDatabases is an empty list.
```

Comment:
Maybe makes more sense to remove it and the user migrates to the new option :)

Reply:
I did not want to break the users, also there is no way of specifying ATM that `$USER` with a program fallback should be used with the `initialDatabases` syntax. It is determined at Nix eval time.
In my personal projects I almost always have a static database name locally, so it can be hardcoded for maximum ease of use. This way we don't have to pass the `$USER` everywhere.

Reply:
If it makes sense to deprecate a flag you can do it like:

```
imports = [
    (lib.mkRenamedOptionModule [ "postgres" "enable" ] [ "services" "postgres" "enable" ])
  ];
```